lib/mrtrix3/dwi2mask/fslbet.py

```diff
@@ -24,12 +24,12 @@ def usage(base_parser, subparsers): #pylint: disable=unused-variable
   parser.set_author('Warda Syeda (wtsyeda@unimelb.edu.au) and Robert E. Smith (robert.smith@florey.edu.au)')
   parser.set_synopsis('Use the FSL Brain Extraction Tool (bet) to generate a brain mask')
   parser.add_citation('Smith, S. M. Fast robust automated brain extraction. Human Brain Mapping, 2002, 17, 143-155', is_external=True)
-  parser.add_argument('input',  help='The input DWI series')
-  parser.add_argument('output', help='The output mask image')
+  parser.add_argument('input', type=app.Parser.ImageIn(), help='The input DWI series')
+  parser.add_argument('output', type=app.Parser.ImageOut(), help='The output mask image')
   options = parser.add_argument_group('Options specific to the \'fslbet\' algorithm')
   options.add_argument('-bet_f', type=float, help='Fractional intensity threshold (0->1); smaller values give larger brain outline estimates')
   options.add_argument('-bet_g', type=float, help='Vertical gradient in fractional intensity threshold (-1->1); positive values give larger brain outline at bottom, smaller at top')
-  options.add_argument('-bet_c', nargs=3, metavar='<x y z>', help='Centre-of-gravity (voxels not mm) of initial mesh surface')
+  options.add_argument('-bet_c', type=app.Parser.SequenceFloat(), nargs=3, metavar='<x y z>', help='Centre-of-gravity (voxels not mm) of initial mesh surface')
```

Comment:
If a voxel position rather than real-space position, metavar should instead be ijk.
Also needs to be either 3 float-type arguments, or one floating-point sequence-type argument.